src/os/FileStore.cc

```diff
@@ -1891,8 +1915,16 @@ void FileStore::_do_op(OpSequencer *osr, ThreadPool::TPHandle &handle)
     dout(5) << "_do_op done stalling" << dendl;
   }
 
-  osr->apply_lock.Lock();
-  Op *o = osr->peek_queue();
+  do {
+    osr->apply_lock.Lock();
+    o = osr->peek_queue();
+    if (o->flags & OP_SKIP_WBTHROTTLE == 0) {
+      osr->apply_lock.Unlock();
+      wbthrottle.throttle();
```

Comment:
@yuyuyu101 . The function of wbthrottle is to control page cache of kernel. So it should not control no-write ops. Further, we already throttle_ops & throttle_bytes. As for benefit, this depend on workload. IIRR, for example mds store file entry w/  omap operation.